src/pages/PullRequestPage/PullCoverage/routes/CommitsTab/CommitsTable/CommitsTable.spec.tsx

```diff
@@ -292,6 +349,30 @@ describe('CommitsTable', () => {
       const coverageColumn = await screen.findByText('89.00%')
       expect(coverageColumn).toBeInTheDocument()
     })
+
+    describe('bundle analysis is enabled', () => {
+      it('renders bundle analysis column', async () => {
+        const { queryClient } = setup({ bundleAnalysisEnabled: true })
+        render(<CommitsTable />, {
+          wrapper: wrapper(queryClient),
+        })
+
+        const bundleAnalysis = await screen.findByText(/Upload: ✅/)
+        expect(bundleAnalysis).toBeInTheDocument()
+      })
+    })
+
+    describe('bundle analysis is disabled', () => {
+      it('does not render bundle analysis column', async () => {
+        const { queryClient } = setup({ bundleAnalysisEnabled: false })
+        render(<CommitsTable />, {
+          wrapper: wrapper(queryClient),
+        })
+
+        const bundleAnalysis = screen.queryByText('Upload: &#x274B;')
```

Comment:
is this an emoji also? Is it definitely correct? The ✅ renders in the PR diff but this one does not, just wondering why.

Reply:
Nice catch, was off by one hex value 🤦